packages/waku/src/router/fs-router.ts

```diff
@@ -95,11 +95,11 @@ export function unstable_fsRouter(
           .split('/')
           .filter(Boolean);
         // must come before isIgnoredPath check to include slices from inside _components
-        if (pathItems.at(-1)?.startsWith('_slice')) {
+        if (pathItems.at(-1)?.startsWith('_slice-')) {
           createSlice({
             component: mod.default,
             render: 'dynamic',
-            id: pathItems.at(-1)!.slice('_'.length),
+            id: pathItems.at(-1)!.slice('_slice-'.length),
```

Comment:
@tylersayshi If we were to modify the code, it should be a separate PR, because merge timings are different.

Reply:
created #1592 separately